recipes/clangdev/build.sh

```diff
@@ -0,0 +1,11 @@
+mkdir build
+cd build
+
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DBUILD_SHARED_LIBS=ON \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  ..
+cmake --build . -- -j 8
```

Comment:
Don't trust the `8` here as we never know how many cores the CIs will provide. If you are feeling brave you can try the `CPU_COUNT` env var which should use the last known number of cores for each CI.